tests/test_qiskit_device.py

```diff
@@ -82,37 +100,37 @@ class TestSupportForV1andV2:
     """Tests compatibility with BackendV1 and BackendV2"""
 
     @pytest.mark.parametrize(
-        "backend",
+        "dev_backend",
```

Comment:
pylint complained or done for better clarity?

Reply:
Yep, `pylint` complained about redefining variable/object `dev` from the above.